common/thread/SignalThread.cpp

```diff
@@ -112,11 +112,11 @@ void* SignalThread::Run() {
   return NULL;
 }
 
+#ifndef _WIN32
 /**
  * @brief Add the signals we're interested in to the sigset.
  */
 bool SignalThread::AddSignals(sigset_t *signals) {
-#ifndef _WIN32
```

Comment:
FWIW sigset_t must be defined on the other flavours of Windows/using pthreads. The problem with ifndefing the whole function is you have to fix up every bit of code. It might make more sense to define sigset_t if it doesn't exist.